src/routers/peers.py

```
@@ -239,25 +240,22 @@ async def get_working_representation(
 
 @router.post(
     "/{peer_id}/search",
-    response_model=Page[schemas.Message],
+    response_model=list[schemas.Message],
     dependencies=[
         Depends(require_auth(workspace_name="workspace_id", peer_name="peer_id"))
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)